doc/ref_cert/RC1/chapters/chapter02.md

```diff
@@ -96,8 +96,8 @@ The Infrastructure Profile Catalog contains the following attributes:
   * A particular set of options is an instance type
   * Compute flavors: .tiny, .small etc as defined in [RM §4.2.1.1](../../../ref_model/chapters/chapter04.md#4211-predefined-compute-flavours)
 * NFVI performance profiles, for which NFVI validations will support and be verified against, are defined as basic and network intensive. Details for each of these profiles can be found in [RM §2.3](../../../ref_model/chapters/chapter02.md#2.3).
-<p align="center"><img src="../figures/RC_NFVI_Profiles.png" alt="NFVI Profiles" title="NFVI Profiles" width="100%"/></p>
-<p align="center"><b>Figure 2-2:</b> NFVI Profiles</p>
+
```

Comment:
The NFVI Profile and associated diagram needs to be updated to the current profiles.

Reply:
Agree. It has to be updated in a second change
https://github.com/cntt-n/CNTT/issues/2611